src/DataLayer/utils/DummyDataSource.ts

```diff
@@ -23,6 +76,25 @@ export class DummyDataSource<T> implements IDataSource<T> {
         return Promise.reject('DummyDataSource reject');
     }
 
+    public getPage(
+        resultSpec: AFM.IResultSpec,
+        // tslint:disable-next-line:variable-name
+        _offset: number[],
```

Comment:
if variable is not used for tslint is enough to write `_: number[]` without using inline comment for disabling tslint check

Reply:
My IDE still has a problem with this. So I leave the comment in place